pkg/commoncfg/mysql_test.go

```diff
@@ -0,0 +1,80 @@
+package commoncfg
```

Comment:
package should be `commoncfg_test` instead of `commoncfg`

Reply:
I prefer adding tests to the same package to be able to examine the state of even private package objects/object fields if needed.
Is there any strict recommendation to have tests in a separate package?